src/Client/app/navigation-header/NavBar.tsx

```
@@ -75,25 +62,23 @@ export default function Component() {
                     <Divider className="mb-5" />
                   </DropdownItem>
                   <DropdownItem key="profile">
-                    <Link
-                      href="/myprofile"
+                    <button onClick={()=>router.push('/myprofile')}
                       className="flex items-center gap-2"
                       style={{ color: 'inherit', textDecoration: 'none' }}
                     >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)